Podcast/FacebookFriendsCollectionViewCell.swift

```diff
@@ -101,7 +101,7 @@ class FacebookFriendsCollectionViewCell: UICollectionViewCell {
     func configureForUser(user: User) {
         imageView.setImageAsynchronouslyWithDefaultImage(url: user.imageURL, defaultImage: #imageLiteral(resourceName: "person"))
         titleLabel.text = user.fullName()
-        detailsLabel.text = "\(user.numberOfFollowers.shortString()) Followers"
+        detailsLabel.text = user.numberOfFollowers == 1 ? "\(user.numberOfFollowers.shortString()) follower" : "\(user.numberOfFollowers.shortString()) followers"
```

Comment:
I'm being picky but we can do `detailsLabel.text = user.numberOfFollowers.shortString() + ( user.numberOfFollowers == 1 ? "follower" :  "followers")`